GroupProject/Startup.cs

```diff
@@ -8,6 +8,8 @@
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using Microsoft.EntityFrameworkCore;
+using GroupProject.Data;
```

Comment:
This was added automatically when the model was created. The idea is that they will be able to store data and not only display it.